### PR TITLE
[lldb] When Swift/C++ is enabled, let Clang imported types format as …

### PR DESCRIPTION
**Pull request by augusto2112**
…Clang types.

Instead of attempting to format Clang types as Swift types, let existing engine to format Clang types do the job.

rdar://100285267

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by augusto2112** [comment]
@swift-ci test

File: lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp [hunk 1]
```diff
@@ -811,6 +811,59 @@ ExtractSwiftTypeFromCxxInteropType(CompilerType type, TypeSystemSwift &ts,
   return {};
 }
 
+
+/// Synthetic child that wraps a value object.
```
Comment (adrian-prantl) [inline lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp:815]: ... that wraps a TypeSystemClang value object?

**Review by adrian-prantl — APPROVED** [review]

File: lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp (hunk 1 above)
Comment (augusto2112) [reply lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp:815]: Actually this is a generic implementation, which is used for both wrapping Clang types in Swift and Swift types in C++

**Pull request merged**